.github/workflows/make_bundle_conda.yml

```diff
@@ -0,0 +1,359 @@
+      artifact-id: ${{ steps.artifact-id.outputs.name }}
+
+    steps:
+      - name: Checkout napari source
+        uses: actions/checkout@v2
+        with:
+          path: napari-source
+          fetch-depth: 0
+
+      - name: Checkout conda-forge feedstock
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          # TODO: Match upstream, not fork!
+          repository: jaimergp/napari-feedstock
+          ref: menuinst-cep
+          path: napari-feedstock
+
+      - name: Create environment.yml
+        shell: bash
+        run: |
+          cat <<EOF > environment.yml
+          name: test
+          channels:
+            - conda-forge
+          dependencies:
+            - conda-smithy
+            - python 3.7
+            - pip
+            - tomlkit
+            - ruamel.yaml
+            - anaconda-client
+          EOF
+
+      - name: Setup Miniconda
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          miniforge-version: latest
+          environment-file: environment.yml
+
+      - name: install napari
+        shell: bash -l {0}
+        working-directory: napari-source
+        run: |
+          set -x
+          pip install -e . --no-deps -vvv
+          version=$(python bundle.py --version)
+          arch=$(python bundle.py --arch)
+          echo NAPARI_VERSION=${version} >> $GITHUB_ENV
+          echo NAPARI_ARCH=${arch} >> $GITHUB_ENV
+
+      - name: Patch and rerender feedstock
+        shell: bash -l {0}
+        working-directory: napari-feedstock
+        run: |
+          set -euxo pipefail
+          # This is the source path once mounted _inside_ the Docker image!
+          source_path="/home/conda/feedstock_root/napari-source"
+
+          sed -i -e "s|{% set version = \".*\" %}|{% set version = \"${NAPARI_VERSION}\" %}|" \
+                 -e "s|  git_url:.*|  path: ${source_path}|" \
+                 -e "s|  git_rev:.*||"  \
+                 recipe/meta.yaml
+
+          conda-smithy rerender
+          git diff
+          mv ${GITHUB_WORKSPACE}/napari-source .
+
+      - name: Run build scripts (PyQt)
+        shell: bash -l {0}
+        working-directory: napari-feedstock
+        env:
+          DOCKER_IMAGE: quay.io/condaforge/linux-anvil-comp7
+          CONFIG: linux_64_qt_bindingspyqt # filename under feedstock's .ci_support/
+          CI: github_actions
+        run: |
+          export CONDA_BLD_PATH="${{ github.workspace }}${{ env.conda_bld_suffix }}"
+          ./.scripts/run_docker_build.sh
+
+      - name: Run build scripts (PySide) # filename under feedstock's .ci_support/
+        shell: bash -l {0}
+        working-directory: napari-feedstock
+        env:
+          DOCKER_IMAGE: quay.io/condaforge/linux-anvil-comp7
+          CONFIG: linux_64_qt_bindingspyside2
+          CI: github_actions
+        run: |
+          export CONDA_BLD_PATH="${{ github.workspace }}${{ env.conda_bld_suffix }}"
+          ./.scripts/run_docker_build.sh
```

Comment:
If I good understand PyQt is built for macOS ARM build? If yes then maybe these steps should be conditional?

Reply:
We build the nightlies for both backends, as `noarch` packages. It's true that for the installer we primarily need the `pyside` one, but I figured it would be beneficial to have both as nightlies too in case at some point we need to debug something. That said, it does introduce extra build time so happy to revisit if the team doesn't agree with the time-cost ratio.

Reply:
I think we can see how things work with both and revisit this in a subsequent step. Will open an issue to keep track of it.

Here -> https://github.com/napari/napari/issues/4010